auspice/server/sources.js

```diff
@@ -26,6 +26,12 @@ class Source {
   get baseUrl() {
     throw InvalidSourceImplementation("baseUrl() must be implemented by subclasses");
   }
+  static isGroup() { /* is the source a "nextstrain group"? */
+    return false;
+  }
+  static isPrivate() { /* does the source require authentication? */
+    return false;
+  }
```

Comment:
The `isPrivate()` methods are duplicative. Calling `visibleToUser()` or `visibleToUser(null)` is valid and used elsewhere in the codebase to see if a Source is visible to an anonymous user.

Reply:
How can I use `visibleToUser` (or another method) to know whether a group (source) is visible to a user because it is public or because it's private and the user is authenticated? I could check for it being an instance of `PrivateGroupSource` but this seems more complex than the solution I implemented.

Reply:
👍 resolved